docs/sectionlist.md

```diff
@@ -267,19 +267,19 @@ This may improve scroll performance for large lists.
 
 Rendered at the bottom of each section.
 
-| Type                                                                 |
-| -------------------------------------------------------------------- |
-| (info: {section: [Section](sectionlist#section)}) => element, `null` |
+| Type                                                                      |
+| ------------------------------------------------------------------------- |
+| `md (info: {section: [Section](sectionlist#section)}) => element \| null` |
```

Comment:
WDYT about using `｜` ([fullwidth vertical line](https://www.compart.com/en/unicode/U+FF5C)) instead of proper vertical line (pipe) character for those cases? The only difference is that proposed character is a monospace one, so paddings are a bit larger.

<img width="846" alt="Screenshot 2023-08-17 at 16 36 42" src="https://github.com/facebook/react-native-website/assets/719641/75752eb2-7046-429e-9131-5546aa6c0078">